libs/langchain/langchain/agents/format_scratchpad/tools.py

```diff
@@ -10,10 +10,10 @@
 
 from langchain.agents.output_parsers.tools import ToolAgentAction
 
+ToolMessagesFormatter = Callable[[Sequence[Tuple[AgentAction, str]]], List[BaseMessage]]
```

Comment:
```suggestion
```

Reply:
Suggesting to either inline the type in the function OR move it to base.py and update the type name to be better reflect what it does (match the variable name)